umbraco/Umbraco-CMS
# Fixed Group Headings in Content

### Prerequisites


If there's an existing issue for this PR then this fixes https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/44

### Description
When a group is created on a document type - in the content view the name of the group appears at the top of the group.  

![image](https://user-images.githubusercontent.com/8383558/179263913-d0f96d56-80ba-45e1-a4a8-b9d577f10450.png)

This is styled to appear as a heading - but it has no correspending H tag or role=heading in the html - this means it does not get announced as a heading to assistive technology.  The impact is that those unable to visually perceive the text will be unaware that they are in a group entitled 'Content'.  This is the same for any new group created in the same way:

![image](https://user-images.githubusercontent.com/8383558/179265876-6bf70774-0cab-4f86-a2d8-29824b2ab1ad.png)


Therefore I have submitted a PR that fixes this issue by adding a H2 tag to the group header. H2 because if the associated PR for [issue 42](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/42) is accepted then it makes sense that the group headings are the next level after the H1 that should be the overall heading for the page (eg content)

![image](https://user-images.githubusercontent.com/8383558/179267479-e28dc62b-191e-4bb7-88ba-1bd6b600abca.png)

Further I have attached additional styling to the section header to match it's previous style so there is minimal impact upon how the groups look visually.

In order to do this - the CSS properties need to be removed from the parent element and placed upon the H2 within.  Futher the original CSS also had an absolute value for the font-size.  This has now been amended to use the same .Less variable that is used elsewhere on the site.

**.umb-group-panel__header** {
  padding: 12px 20px;
  display: flex;
  align-items: center;
  justify-content: space-between;
  color: @grayDarker;
  border-bottom: 1px solid @gray-9;
}

**.umb-group-panel__header h2** {
  font-size: @fontSizeMedium;
  font-weight: bold;
}

### Testing

In order to test this:

- Create a dummy site with a basic document type to represent a content or home page
- Add some groups such as 'Content' and 'Header' as per images above
- Go to the content editing section of the backoffice
- Create a new page and use whichever browser's 'inspect' tool over the content groups.  These should now be H2
- This also works correctly with a screen reader: note - in order for the headings to be nested correctly this PR should only be merged after [issue 42](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/42)
